tests/test_default_gaussian.py

```diff
@@ -76,7 +77,7 @@ def test_sample_exception(self):
         def circuit():
             return qml.sample(qml.NumberOperator(0), 10)
 
-        with self.assertRaisesRegex(NotImplementedError, "Sampling is not supported in default.gaussian"):
+        with self.assertRaisesRegex(NotImplementedError, "Sampling for this observable is not implemented."):

```

Comment:
Will need to update this error message if the above suggestion is accepted

Reply:
Agreed. I am not sure what this `.assertRaisesRegex` does though

Reply:
It is the unittest equivalent of `pytest.raises()`